### ISSUE TITLE
Incorrect logged time in jira

### ISSUE THREAD

**Issue by dimitar-petrov** [ahungry/org-jira#144]
If a issue is updated with multiple lines of logged time the logged time in jira is not calculated correctly:

The problem is visible bellow. Jira is updating jiraissue.timespent when api request is received, but looks like there might be problem with async updates.

Is org-jira updating the tracked time asynchroneously?

```
#+BEGIN_SRC sql :engine postgresql :dbhost colossus :database jira :dbuser jira :dbpassword ******
  select a.id, a.summary, a.timespent,b.sum from jiraissue a inner join (
    select issueid, sum(timeworked) from worklog where issueid in (
      select id from jiraissue) group by issueid) b on a.id = b.issueid where a.timespent != b.sum;
#+END_SRC

#+RESULTS:
|    id | summary                                                     | timespent |   sum |
|-------+-------------------------------------------------------------+-----------+-------|
| 10007 | Finalize RAID configuration of colossus and invictus        |      5400 | 13320 |
| 10104 | In ***** is throwing KeyError                               |       960 |  1860 |

```
Similar issue is mentioned here.

https://community.atlassian.com/t5/Jira-Software-questions/Time-Tracking-quot-Logged-quot-shows-wrong-value/qaq-p/647203

Is someone else facing this problem and what is a good way to mitigate it.

**Comment by marc0s** [comment]
Just to let you know, I'm facing this very same issue.

**Comment by ahungry** [comment]
Is this a problem with Jira failing to calculate a time spent sum on their end?

org-jira pushes lots of stuff async. 

When I push multiple time entries, I see them all listed on the Jira side.  I do not usually look at a SUM total on Jira side though.

**Comment by dimitar-petrov** [comment]
> Is this a problem with Jira failing to calculate a time spent sum on their end?

DP: Yes. I believe so.

**Comment by shivan** [comment]
Is there a bugfix from atlassian for that? Experiencing the same here with another application using jira api.

**Comment by jack-michaud** [comment]
I have a PR in review #246 that fixes this issue on the org-jira side. 

The explanation of this fix is in this issue: #247 
**TLDR**: worklogs should be updated/created synchronously to keep the time tracking accurate. 

Hopefully this hits mainline!

**Comment by ahungry** [comment]
Should be taken care of via https://github.com/ahungry/org-jira/pull/246
